www/front_src/src/Provider/index.tsx

```diff
@@ -39,7 +39,7 @@ const AppProvider = (): JSX.Element | null => {
     Promise.all([
       getUser(userEndpoint),
       getTranslations(translationEndpoint),
-      getAcl(aclEndpoint),
+      getAcl('http://localhost:5000/mock/acl'),
```

Comment:
Isn't it better to capture this information in a configuration or constant to be reused?